src/common/btree.c

```diff
@@ -3658,6 +3665,10 @@ dbtree_iter_delete(daos_handle_t ih, void *args)
 	if (rc != 0)
 		return rc;
 
+	/** The probed entry is not committed so return -DER_TX_BUSY */
```

Comment:
I don't see why an aborted entry can't be removed?

Reply:
You are right.   This needs to be for in-progress entries.   We need another way to determine if an entry is aborted.   I don't think the dtx_status stuff is working for single value